_posts/2020-02-03-release-1.10.0.md

```diff
@@ -0,0 +1,187 @@
+
+<span>
+	<center>
+	<img vspace="8" style="width:100%" src="{{site.baseurl}}/img/blog/2020-02-03-release-1.10.0/flink_1.10_zeppelin.png" />
+	</center>
+</span>
+
+In particular, these changes make it much easier to programmatically use Flink in downstream frameworks — for example, Apache Beam or Zeppelin interactive notebooks — by providing users with a unified entry point to Flink. For users working with Flink across multiple target environments, the transition to a configuration-based execution process also significantly reduces boilerplate code and maintainability overhead.
+
+### Native Kubernetes Integration (Beta)
+
+For users looking to get started with Flink on a containerized environment, deploying and managing a standalone cluster on top of Kubernetes requires some upfront knowledge about containers, operators and environment-specific tools like `kubectl`.
+
+In Flink 1.10, we rolled out the first phase of **Active Kubernetes Integration** ([FLINK-9953](https://jira.apache.org/jira/browse/FLINK-9953)) with support for session clusters (with per-job planned). In this context, “active” means that Flink’s ResourceManager (`K8sResMngr`) natively communicates with Kubernetes to allocate new pods on-demand, similar to Flink’s Yarn and Mesos integration. Users can also leverage namespaces to launch Flink clusters for multi-tenant environments with limited aggregate resource consumption. RBAC roles and service accounts with enough permission should be configured beforehand.
+
+<span>
+	<center>
+	<img vspace="8" style="width:75%" src="{{site.baseurl}}/img/blog/2020-02-03-release-1.10.0/flink_1.10_nativek8s.png"/>
+	</center>
+</span>
+
+As introduced in [Unified Logic For Job Submission](#unified-logic-for-job-submission), all command-line options in Flink 1.10 are mapped to a unified configuration. For this reason, users can simply refer to the Kubernetes config options and submit a job to an existing Flink session on Kubernetes in the CLI using:
+
+```bash
+./bin/flink run -d -e kubernetes-session -Dkubernetes.cluster-id=<ClusterId> examples/streaming/WindowJoin.jar
+```
+
+If you want to try out this preview feature, we encourage you to walk through the [Native Kubernetes setup]({{ site.DOCS_BASE_URL }}flink-docs-release-1.10/ops/deployment/native_kubernetes.html), play around with it and share feedback with the community.
+
+### Table API/SQL: Production-ready Hive Integration
+
+Hive integration was announced as a preview feature in Flink 1.9. This first implementation allowed users to persist Flink-specific metadata in Hive Metastore using SQL DDL, call UDFs defined in Hive and use Flink for reading and writing Hive tables. However, some further development was still required to really bring production-ready Hive integration to Flink.
+
```

Comment:
```
#### Compatibility with More Hive Versions

Flink 1.9 only natively supports Hive 1.2.1 and 2.3.4, and relies on Hive's own compatibility for the rest of Hive 1.x.x and 2.x.x versions.

Flink 1.10 introduces [native compatibility for all the Hive 1.x.x, 2.x.x, and 3.x.x versions]({{ site.DOCS_BASE_URL }}flink-docs-release-1.10/dev/table/hive/#supported-hive-versions).

#### Enhanced Experience to Persist Flink Tables in Hive Metastore

In Flink 1.9, we introduced a new beta feature to allow users persist their Flink tables created from DDL into Hive Metastore. It solves users' pain point of unable to reuse Flink tables created from DDL across sessions and having to re-create those tables every time.

We made the feature more robust and the experience more pleasant in Flink 1.10 and made it compatible with all 1.x.x, 2.x.x, and 3.x.x Hive Metastore versions.
```

Reply:
How relevant would you consider this in comparison to what is already covered for Hive? It seems really prominent in the first few paragraphs of entering the Hive Integration documentation. 

I'm afraid the blogpost is getting too exhaustive/extensive and would prefer to leave this kind of details to a follow-up blogpost dedicated to the Flink-Hive integration. What do you think?

Reply:
Sounds reasonable. I'll incorporate this into the 1st paragraph

Reply:
Thanks, @bowenli86! @MarkSfik will get in touch with you about the blogpost.